### DISCUSSION TITLE
How to get transform type, especially if it is rotation before it starts?

### DISCUSSION THREAD

**Discussion by lanyusan** [konvajs/konva#1221] [Q&A] [answered]
My  use case is that for a shape group, if it is rotation,  wrapping rect and entire group rotates, while for resizing, only wrapping rect resizes.

Is there a way to tell the transform type so app can dynamically change tranform nodes?

Thanks.

**Comment by lavrton** [comment, marked as answer]
```js
transformer.on('transformstart', () => {
  const isRotating = transformer.getActiveAnchor() === 'rotater';
});
```

**Reply by lanyusan** [reply]
Thanks.
